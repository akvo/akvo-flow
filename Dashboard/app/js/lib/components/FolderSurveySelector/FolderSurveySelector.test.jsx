describe('=== FolderSurveySelector Tests ===', () => {
  it('+++ runs a test', () => {
    expect(true).toEqual(true);
  });
});
