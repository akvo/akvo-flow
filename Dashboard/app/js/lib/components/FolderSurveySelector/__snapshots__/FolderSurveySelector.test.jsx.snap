// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderSurveySelector Tests +++ calls for new levels and update state when option is selected 1`] = `
<div>
  <div>
    <select
      data-testid="folder-survey-select-0"
    >
      <option
        value="0"
      >
        Choose a folder or survey
      </option>
      <option
        value="153142013"
      >
        Folder with few large data sets
      </option>
    </select>
    <select
      data-testid="folder-survey-select-1"
    >
      <option
        value="0"
      >
        Choose a folder or survey
      </option>
      <option
        value="152342023"
      >
        BAR-handpump
      </option>
    </select>
  </div>
</div>
`;

exports[`FolderSurveySelector Tests +++ renders <snapshot> 1`] = `
<div>
  <div>
    <select
      data-testid="folder-survey-select-0"
    >
      <option
        value="0"
      >
        Choose a folder or survey
      </option>
      <option
        value="153142013"
      >
        Folder with few large data sets
      </option>
    </select>
  </div>
</div>
`;
